less/index.less

```diff
@@ -53,7 +53,187 @@ table {
     box-sizing: border-box;
 }
 
+/* Variables */
+
+@nice-grey: #f0e4c5;
```

Comment:
Nice use of variables, but there's still room for you to use it more. Look at the CSS properties you are using multiple times and you can make them into variables